.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+# Pull Request Template
```

Comment:
I would remove the pull request template and just make both of these their own header. It should be obvious that it's a template from the fact that they're seeing it in the PR description

Reply:
Got it